### ISSUE TITLE
Delta LM model checkpoint

### ISSUE THREAD

**Issue by ndenStanford** [microsoft/unilm#1002]
I have explored the possibility of using the [DeltaLM](https://github.com/microsoft/unilm/tree/master/deltalm) model in performing multilingual abstractive summarization. The [paper](https://arxiv.org/pdf/2106.13736.pdf) has demonstrated a SOTA performance in this task. Would you mind letting me know if the trained checkpoint for such a task is available? I appreciate your attention to this matter.

**Comment by shumingma** [comment]
Hi, thanks for the interest in our work. Unfortunately, we don't have any plan to release the checkpoints finetuned on the downstream tasks. Yet, both the pretrained DeltaLM and the dataset in our paper are public available, so it should be not difficult to reproduce the results in the paper.

**Comment by yugaljain1999** [comment]
Hey @shumingma I have one small query though, Can you pls tell me where I can find name of all languages(100+ as per paper) ?
As I want to finetune deltaLM for translation of below languages..

German
Russian Spanish
Chinese
French
Finnish
Swedish
Polish
Dutch
Portuguese
Slovak
Hungarian
Indonesian
Italian
Danish
Romanian
Serbian
Norwegian
Czech
Slovenian	

Looking forward to your response..
Thanks

**Comment by shumingma** [comment]
Hi @yugaljain1999, DeltaLM supports the same languages as InfoXLM. You can find those languages in the appendix of [InfoXLM paper](https://arxiv.org/abs/2007.07834).

**Comment by yugaljain1999** [comment]
@shumingma Okay got that. Just one more query, I am struggling to write proper inference script for DeltaLM model inference.
So can you pls refer  what command we should use to test inference one to one of DeltaLM on input as plain text and output as translation? Like you mentioned for evaluation on test set.

It would be really appreciable.
Thanks
